### ISSUE TITLE
Bug in facenet-pytorch 2.0

### ISSUE THREAD

**Issue by ArtanisCV** [timesler/facenet-pytorch#52]
Thanks for your GPU implementation for MTCNN! It works great!! The speed improvement is significant.

There is a small issue:
https://github.com/timesler/facenet-pytorch/blob/master/models/utils/detect_face.py#L49
https://github.com/timesler/facenet-pytorch/blob/master/models/utils/detect_face.py#L124
It seems that torch.cat / torch.stack does not have an argument named 'axis'. Should it be 'dim' ?

**Comment by timesler** [comment]
Thanks for spotting this @ArtanisCV. I rushed the release a bit to make it available for the current DeepFakes competition. It's interesting that I didn't run into any testing errors in the CI tests for this. Pytorch must have quietly added support for numpy-like argument names in a recent version. Either way, I've made the correction to `dim`

**Comment by timesler** [comment]
I will get the update onto pypi tomorrow
